extras/videoDrivers/SDL2/VideoSDL2.cpp

```diff
@@ -346,7 +351,7 @@ bool VideoSDL2::MessageLoop()
                 break;
             }
             case SDL_MOUSEBUTTONDOWN:
-                mouse_xy.pos = Position(ev.button.x, ev.button.y);
+                mouse_xy.pos = Position(GetGuiScale().screenToViewCoords(Position(ev.button.x, ev.button.y)));
```

Comment:
Originally, this was written as `mouse_xy.pos = GetGuiScale().screenToViewCoords(Position(ev.button.x, ev.button.y));` when `screenToViewCoords()` had a template parameter for the return type that was defaulted to `Position`.
In other places, one could write `GetGuiScale().screenToViewCoords<Extent>(...)`, for example. I removed the template parameter because it mirrored the style of the rest of the code.
The upside of the template parameter here is fewer parentheses, which improves readability.
It would be easy to revert the commit that introduced this change.

Reply:
I've removed c07bd33 (f0700f4c1) after seeing other places in the code use the same template-based cast pattern. (And `iround()` uses it too, of course.)